pkg/cloudprovider/provider/cherryservers/provider.go

```diff
@@ -0,0 +1,333 @@
+package cherryservers
+
+import (
```

Comment:
Can we reorganize the imports to follow the order of:
* stdlib
* external dependencies
* machine-controller packages
* `k8s.io/*`
* `sigs.k8s.io/*`

Here is how it looks like for other packages

https://github.com/kubermatic/machine-controller/blob/9200b1809b024a90246d6496804409df6474f328/pkg/cloudprovider/provider/aws/provider.go#L19-L49

Reply:
Fixed